From 1bbb884b6fbe4502063550e9a22428cafe0453b6 Mon Sep 17 00:00:00 2001
From: Son Luong Ngoc <sluongng@gmail.com>
Date: Thu, 1 Jun 2023 13:11:02 +0200
Subject: [PATCH] go_netrc: switch back to gazelle style import

This enables other projects to depend on bazelisk using a gazelle
generated go_repository for com_github_bgentry_go_netrc, instead of
having to use the archive strip_prefix go_repository that Bazelisk
handcrafted.
---
 WORKSPACE      | 7 +++----
 httputil/BUILD | 2 +-
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index 4304243..73033ce 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -44,10 +44,9 @@ go_repository(
 
 go_repository(
     name = "com_github_bgentry_go_netrc",
-    importpath = "github.com/bgentry/go-netrc/netrc",
-    urls = ["https://github.com/bgentry/go-netrc/archive/9fd32a8.zip"],
-    type = "zip",
-    strip_prefix = "go-netrc-9fd32a8b3d3d3f9d43c341bfe098430e07609480/netrc",
+    importpath = "github.com/bgentry/go-netrc",
+    sum = "h1:xDfNPAt8lFiC1UJrqV3uuy861HCTo708pDMbjHHdCas=",
+    version = "v0.0.0-20140422174119-9fd32a8b3d3d",
 )
 
 go_rules_dependencies()
diff --git a/httputil/BUILD b/httputil/BUILD
index d48f290..4c7ac89 100644
--- a/httputil/BUILD
+++ b/httputil/BUILD
@@ -13,7 +13,7 @@ go_library(
     importpath = "github.com/bazelbuild/bazelisk/httputil",
     visibility = ["//visibility:public"],
     deps = [
-        "@com_github_bgentry_go_netrc//:go_default_library",
+        "@com_github_bgentry_go_netrc//netrc:go_default_library",
         "@com_github_mitchellh_go_homedir//:go_default_library",
     ],
 )
